browserify.js

```diff
@@ -1,14 +1,29 @@
 // Browserify wrapper for webkit-assign that returns a stream.
 //
-// A wrapper is necessary because webkitAssign(filename) in index.js checks 
-// whether the first parameter is a string. Since Browserify passes the filename 
-// as the first param, webkitAssign calls transformCode() which returns a string 
+// A wrapper is necessary because webkitAssign(filename) in index.js checks
+// whether the first parameter is a string. Since Browserify passes the filename
+// as the first param, webkitAssign calls transformCode() which returns a string
 // instead of a stream expected by Browserify.
 
 'use strict';
 
 var webkitAssign = require('./index');
+var through = require('through2');
+var _ = require('lodash');
 
-module.exports = function () {
-    return webkitAssign();
+var defaultOptions = { ext: ['.js'] };
+
+module.exports = function (file, options) {
+    // Merge options with default options
+    _.defaults(options, defaultOptions);
+    // Check if it matches accepting extention
+    var match = _.any(options.ext, function(opt) {
```

Comment:
Please add a space after the `function` keyword and call the variable `extension`.